src/components/globalConfigurations/GlobalConfiguration.tsx

```diff
@@ -177,7 +190,7 @@ function Body({ getHostURLConfig, checkList, serverMode}) {
         }} />
         <Route path={URLS.GLOBAL_CONFIG_DOCKER} render={(props) => {
             return <div className="flexbox h-100">
-                <Docker {...props} />
+                <Docker {...props} getAppCheckList={getAppCheckList} handleChecklistUpdate={handleChecklistUpdate}/>
```

Comment:
Do we need to pass getAppCheckList as prop?

Reply:
NO removing